### ISSUE TITLE
Document how you're supposed to use minimalcss

### ISSUE THREAD

**Issue by peterbe** [peterbe/minimalcss#93]
Generally the idea is that you replace your bloated external style sheets with one block of minimal CSS. That can either be injected as a new external .css or inlined. But it's generally a good idea to do something like this:

Before:
```html
<link rel="stylesheet" href="bloatedframework.css">
```

After:
```html
<link rel="stylesheet" href="builds/minimal.css">
<link rel="preload" href="bloatedframework.css" as="style" onload="this.rel='stylesheet'">
```

The reason why it's a good idea to still lazy load in that `bloatedframework.css` is because `minimalcss` can only figure out the minimal CSS on first page load (including XHR) before the user starts interacting and using the pointer to cause the DOM to change. 

All of that needs to be explained! Ideally quite early in the README to grab peoples attention.

Also, it should probably link to [loadCSS](https://github.com/filamentgroup/loadCSS) which has a good polyfill solution to lazy loading CSS. The code example above doesn't work in all browsers.

**Comment by peterbe** [comment]
Bonus points if we can have some `gh-pages` examples that demonstrate this with URLs you can throw into webpagetest.

**Comment by stereobooster** [comment]
Do not forget 

```
<noscript>
  <link rel="stylesheet" href="bloatedframework.css">
</noscript>
```

**Comment by peterbe** [comment]
Yep, I think that's part of the `loadCSS` solution. It's what I use on https://songsear.ch

**Comment by stereobooster** [comment]
One of my examples https://github.com/stereobooster/react-snap/blob/master/doc/emotion-site-optimization.md 

We can use any big boostrap page for demo, for example https://startbootstrap.com/template-overviews/sb-admin-2/
